pulseapi/users/adapter.py

```diff
@@ -82,7 +82,7 @@ class PulseSocialAccountAdapter(DefaultSocialAccountAdapter):
         2. Handle the transition from the old auth system to "upgrade" existing
         accounts to the new allauth system.
     """
-    def is_open_for_signup(self, request):
+    def is_open_for_signup(self, request, socialaccount):
```

Comment:
as per https://github.com/pennersr/django-allauth/blob/198d356be0cc24081340b803624a2ae410c62155/docs/advanced.rst#creating-and-populating-user-instances